src/modules/roc_sdp/parser.rl

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include "roc_core/log.h"
+#include "roc_core/panic.h"
+#include "roc_core/array.h"
+#include "roc_core/heap_allocator.h"
+#include "roc_core/shared_ptr.h"
+#include "roc_sdp/session_description.h"
+#include "roc_sdp/media_type.h"
+#include "roc_sdp/media_transport.h"
+
+namespace roc {
+namespace sdp {
+
+%%{
+    machine parse_sdp;
+    write data;
+}%%
+
+bool parse_sdp_imp(const char* str, SessionDescription& result);
```

Comment:
We should either declare it static or, better, define it in an anonymous namespace.